### ISSUE TITLE
Fund Raising with Gitcoin 

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/Communication#88]
TL;DR
Walk through the giveth repo and fill out [https://gitcoin.co/requests](https://gitcoin.co/requests) for support in funding bounties :-D 
----
From Mitch:

Just funded 100 DAI for https://github.com/Giveth/giveth-dapp/issues/474 :) (may be a bit rich for funding but happy to get things started up for ECF fund)

If you have some time, would appreciate if you could walk through the giveth repo and fill out https://gitcoin.co/requests for issues that are relevant to the community along with DAI amounts that you think are appropriate... very happy to do some bottom-up funding for this great project!

thanks :)
---

**Comment by anshumanv** [comment]
On it!

**Comment by csggene3** [comment]
I can work from the bottom up if anshumanv wants to work from the top down =)

**Comment by krrisis** [comment]
How's this one going guys?

**Comment by csggene3** [comment]
I'd be happy to make progress on this today but will still need individual recommendations as to amounts up for reward. Who should I ask on each issue? I know Griff is probably busy...

**Comment by csggene3** [comment]
https://github.com/Giveth/giveth-dapp/issues/488 - being posted at 30DAI

**Comment by csggene3** [comment]
https://github.com/Giveth/giveth-dapp/issues/475 - being posted at 70 DAI

**Comment by csggene3** [comment]
I can do these individually here, and save everybody call time if I can just get a thumbs up or thumbs down, then I'll go ahead and fill out the forms for thumbs up accordingly. Could get the majority of the list at least addressed here by today if that works for everyone.

**Comment by GriffGreen** [comment]
Giveth/giveth-dapp#488  - 30 DAI sounds good :-D

#475 I would say 70 DAI

**Comment by mkosowsk** [comment]
Mitch Kosowski reporting in! Thanks Giveth team for your help 👍 

I think it's good to chat ideas here but for my process would be good to run through issues after Giveth has submitted them via Gitcoin Requests :) I'm currently juggling funding 15-20 repos so I won't be able to have a super amount of input into any one issue unfortunately

**Comment by csggene3** [comment]
No problem and sounds good on the after versus before. Since you are so busy, I'll go ahead and run through what has already been posted by you with amounts to keep a complete record in here so you don't have to worry with it. Then ideally we will all be caught up to the same page, and I can start submitting those that have not been addressed yet. This is the only repo I am funding, so I have the time to do things issue by issue if you do not, it is my pleasure to do so. Like I said in the email, well done to you for getting so many of them up there as it is and being ahead of the game. 👍🐭

**Comment by csggene3** [comment]
Gitcoin bot is offering up workers already, but I'll leave it to any of the rest of you to approve them or not.

**Comment by csggene3** [comment]
https://github.com/Giveth/giveth-dapp/issues/206 30 DAI?

**Comment by jessicazartler** [comment]
@krrisis anything I can do to support here? Can you catch me up on this so we can include in our fundraising documentation? Thank you!

**Comment by krrisis** [comment]
this is specifically for dapp issues - we'll talk about it during our chat

**Comment by krrisis** [comment]
All done!
